client/js/libs/handlebars/index.js

```diff
@@ -0,0 +1,18 @@
+"use strict";
+
+module.exports = {
+	colorClass: require("./colorClass"),
+	diff: require("./diff"),
+	equal: require("./equal"),
+	firstLetterUppercase: require("./firstLetterUppercase"),
+	localedate: require("./localedate"),
+	localetime: require("./localetime"),
+	modes: require("./modes"),
+	parse: require("./parse"),
+	roundBadgeNumber: require("./roundBadgeNumber"),
+	slugify: require("./slugify"),
+	ternary: require("./ternary"),
+	tojson: require("./tojson"),
+	tz: require("./tz"),
+	users: require("./users"),
+};
```

Comment:
Yeah, it isn't necessary because with the webpack handlebars loader, we just point it at a folder and it takes all of them and registers them as handlebars helpers.

The same doesn't exist for anything express (that I could find, feel free to find one if it exists). We could write something for it, but meh. The one that exists for express, you have to specify everything specifically. Not ideal, but meh.